### DISCUSSION TITLE
2.8.2

### DISCUSSION THREAD

**Discussion by ryanluker** [ryanluker/vscode-coverage-gutters#346] [General]
https://github.com/ryanluker/vscode-coverage-gutters/milestone/38?closed=1

Removed sentry and telemetry from the extension due to cross extension error captures.

<hr /><em>This discussion was created from the release <a href='https://github.com/ryanluker/vscode-coverage-gutters/releases/tag/v2.8.2'>2.8.2</a>.</em>

**Comment by Prophecy000** [comment]
Hi to everyone in the case of the tree itself the master would have to come in and have a look. Created a bug when you deal with codes that doesn't formulate into the tree . My way of thinking is do you know your roots? I used this phrase when looking at snps in dna etc..
